AMW_business/src/main/resources/liquibase/changesets/base/data/exampleapp/app.data.xml

```diff
@@ -16,32 +16,29 @@
 		</sql>
 	</changeSet>
 
-	<changeSet author="initial-data" id="example-resources-pg" dbms="postgresql">
+	<changeSet author="initial-data" id="example-resources">
```

Comment:
not sure about that, I would rather define the supported dbms explicitly than the "hack" with the precondition